queries/python/highlights.scm

```diff
@@ -13,10 +13,22 @@
 ((identifier) @constant.builtin
  (#match? @constant.builtin "^__[a-zA-Z0-9_]*__$"))
 
+((identifier) @constant.builtin
+ (#vim-match?
+  @constant.builtin
+  ; https://docs.python.org/3/library/constants.html
+  "^(NotImplemented|Ellipsis|quit|exit|copyright|credits|license)$"))
```

Comment:
Can you indent it like that? Lisp convention is usually to have function arguments align with their indentation and to use `;;` when comment refers to next line of the same sexpr.
```scheme
((identifier) @constant.builtin
 (#vim-match?  @constant.builtin
                        ;; https://docs.python.org/3/library/constants.html
                        "^(NotImplemented|Ellipsis|quit|exit|copyright|credits|license)$"))
```

Reply:
Done!